Maple2.Server.DebugGame/Graphics/ImGuiController.cs

```diff
@@ -18,12 +18,12 @@ public enum ImGuiWindowType {
 public class ImGuiController {
     public static readonly ILogger Logger = Log.Logger.ForContext<ImGuiController>();
 
-    public DebugGraphicsContext Context { get; init; }
-    public IWindow? ParentWindow { get; private set; }
-    public IInputContext Input { get; init; }
-    public IntPtr ImGuiContext { get; private set; }
-    public ImGuiWindowType WindowType { get; init; }
-    private List<char> pressedCharacters;
+    private DebugGraphicsContext Context { get; init; }
+    private IWindow? ParentWindow { get; set; }
```

Comment:
Why does this need to be private?

Reply:
It doesn't need to be but it was just editor complaining